forui/lib/src/widgets/header/nested_header.dart

```diff
@@ -0,0 +1,166 @@
+part of 'header.dart';
+
+/// A nested header.
+///
+/// A nested header contains the page's title and actions (including pop navigation).
+/// It is typically used on pages that are not at the root of the navigation stack.
+///
+/// See:
+/// * https://forui.dev/docs/header for working examples.
+/// * [FNestedHeaderStyle] for customizing a header's appearance.
+final class _FNestedHeader extends FHeader {
+  /// The style. Defaults to [FThemeData.headerStyle.nestedStyle].
+  final FNestedHeaderStyle? style;
+
+  /// The title, aligned to the center.
+  final Widget title;
+
+  /// The actions, aligned to the left. Defaults to an empty list.
+  final List<Widget> leftActions;
+
+  /// The actions, aligned to the right. Defaults to an empty list.
+  final List<Widget> rightActions;
+
+  /// Creates a [_FNestedHeader].
+  const _FNestedHeader({
+    required this.title,
+    this.style,
+    this.leftActions = const [],
+    this.rightActions = const [],
+    super.key,
+  }) : super._();
+
+  @override
+  Widget build(BuildContext context) {
+    final style = this.style ?? context.theme.headerStyle.nestedStyle;
+
+    return SafeArea(
+      bottom: false,
+      child: Padding(
+        padding: style.padding,
+        child: Stack(
+          children: [
+            _InheritedActionStyle(
+              style: style.actionStyle,
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  Row(children: leftActions.expand((action) => [action, const SizedBox(width: 10)]).toList()),
+                  Row(children: rightActions.expand((action) => [const SizedBox(width: 10), action]).toList()),
+                ],
+              ),
+            ),
+            Positioned.fill(
+              child: Align(
+                child: DefaultTextStyle.merge(
+                  overflow: TextOverflow.fade,
+                  maxLines: 1,
+                  softWrap: false,
+                  style: style.titleTextStyle,
+                  child: title,
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties
+      ..add(DiagnosticsProperty('style', style))
+      ..add(IterableProperty('leftActions', leftActions))
+      ..add(IterableProperty('rightActions', rightActions));
+  }
+}
+
+/// [_FNestedHeader]'s style.
```

Comment:
We shouldn't reference private types since the links will not work in the generated docs.

```suggestion
/// [FHeader.nested] 's style.
```